docs/uplink/ingress-for-tunnels.md

```diff
@@ -0,0 +1,275 @@
+# Ingress for tunnels
+
```

Comment:
Info message (look up the syntax)

Inlets Uplink is designed to connect customer services to a remote Kubernetes cluster for command and control as part of a SaaS product.

Any tunnelled service can be accessed directly from within the cluster and does not need to be exposed to the public Internet for access.

Beware: by following these instructions, you are exposing one or more of those tunnels to the public Internet.